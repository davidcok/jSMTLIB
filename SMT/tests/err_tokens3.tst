; unclosed string
(set-logic QF_UF)
(assert "asd )
