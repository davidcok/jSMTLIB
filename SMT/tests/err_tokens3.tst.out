success
(assert "asd )
        ^^^^^^^
(error "String literal is not terminated: \"asd )
")
